contracts/libraries/YearnCurveHelpers.sol

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+import "./../interfaces/external/ICurvePool/ICurvePool.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/// @notice Library for Yearn Curve deposit/withdraw
+library YearnCurveHelpers {
+    using SafeERC20 for IERC20;
+
+    /// @dev Get the array of token amount to send to a
+    ///      Curve 2pool to add liquidity
+    /// @param pool The curve 2pool
+    /// @param token The token to remove from the pool
+    /// @param amount The amount of token to remove from the pool
+    /// @return amounts Array of 2 token amounts sorted by Curve pool token indexes
```

Comment:
We named the return variables `amounts` but there is no mention in the code.
Should we remove the name in the natspect ?
What do you think @Yashiru ?